### PR TITLE
Add more documentation for JsonSchemaRenderer

### PR DESCRIPTION
**Pull request by grahammcculloch**
Change-type: patch
Signed-off-by: Graham McCulloch <graham@balena.io>

---

##### Contributor checklist
- [x] I have rebuilt the README with `npm run build:docs`
- [ ] I have regenerated screenshots for any affected components with `npm run generate-screenshots`
- [ ] I have regenerated jest snapshots for any affected components with `npm run jest -- -u`

##### Reviewer Guidelines
- When submitting a review, please pick:
  - '*Approve*' if this change would be acceptable in the codebase (even if there are minor or cosmetic tweaks that could be improved).
  - '*Request Changes*' if this change would not be acceptable in our codebase (e.g. bugs, changes that will make development harder in future, security/performance issues, etc).
  - '*Comment*' if you don't feel you have enough information to decide either way (e.g. if you have major questions, or you don't understand the context of the change sufficiently to fully review yourself, but want to make a comment)
---

### CODE REVIEW

**Comment by grahammcculloch** [comment]
@balena-ci rebase

**Comment by grahammcculloch** [comment]
@balena-ci rebase

File: README.md [hunk 1]
```diff
@@ -1120,6 +1122,356 @@ A component used to render JSON data based on a schema and a UI schema.
...
+    items: {
+      'ui:options': {
+        href: 'https://mysite.com/${source}'
+      }
+    }
+  }
+}
+```
+
+In this example a json-e tranform is run for each item in the `links` array, allowing a unique value of `ui:options.href` to be set for each link in the array.
```
Comment (LucianBuzzo) [inline README.md:1164]: NIT: small typo `tranform`
```suggestion
In this example a json-e transform is run for each item in the `links` array, allowing a unique value of `ui:options.href` to be set for each link in the array.
```

**Review by LucianBuzzo — COMMENTED** [review]
Great work! This is going to make using this component a lot easier for people 👍

File: README.md (hunk 1 above)
Comment (grahammcculloch) [reply README.md]: Woah - eagle-eyes!! Impressive spot (or did you use Grammarly...? 😆). Fixed now.

**Comment by grahammcculloch** [comment]
@balena-ci rebase

**Review by LucianBuzzo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (LucianBuzzo) [reply README.md]: @grahammcculloch :joy: I actually spotted this one "raw"

**Pull request merged**